src/DatePicker/index.js

```diff
@@ -68,7 +70,7 @@ export default class DatePicker {
         <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
 
         <div class="date-picker__calendar" style="display: none">
```

Comment:
Yes!  I haven't used that one, but I think it definitely applies.  Also, throw an id on this for the buttons to `aria-controls`.  Making sure that id is unique per-instance may be tricky though.